exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillRelFactories.java

```diff
@@ -122,4 +127,16 @@ public RelNode createJoin(RelNode left, RelNode right,
     }
   }
 
+  private static class DrillAggregateFactoryImpl implements RelFactories.AggregateFactory {
+
+    @Override
+    public RelNode createAggregate(RelNode input, boolean indicator, ImmutableBitSet groupSet,
+                                   ImmutableList<ImmutableBitSet> groupSets, List<AggregateCall> aggCalls) {
+      try {
+        return new DrillAggregateRel(input.getCluster(), input.getTraitSet(), input, indicator, groupSet, groupSets, aggCalls);
+      } catch (InvalidRelException ex) {
```

Comment:
Done.

Reply:
I have checked that `InvalidRelException` in `DrillAggregateRel` and `DrillAggregateRelBase` constructors declarations is redundant, it can never be thrown. So I suppose it is safe to remove it there and also all `try-catch` blocks connected to it.